docs/compose.md

```diff
@@ -49,8 +49,7 @@ should be used to create a image based on the NGC 21.11 Triton release.
 
 `compose.py` provides `--backend`, `--repoagent` options that allow you to
 specify which backends and repository agents to include in the custom image.
-The `--enable-gpu` flag indicates that you want to create an image that supports
-NVIDIA GPUs. For example, the following creates a new docker image that
+For example, the following creates a new docker image that
```

Comment:
Remove --enable-gpu from the below commands. We don't want any mention of it in docs or --help